torch/testing/_asserts.py

```diff
@@ -102,6 +102,76 @@ def wrapper(actual: Tensor, expected: Tensor, **kwargs: Any) -> Optional[Excepti
     return wrapper
 
 
+def _check_sparse_coo_members_individually(
+    check_tensors: Callable[..., Optional[Exception]]
+) -> Callable[..., Optional[Exception]]:
+    """Decorates strided tensor check functions to individually handle sparse COO members.
+
+    If the inputs are not sparse COO, this decorator is a no-op.
+
+    Args:
+        check_tensors (Callable[[Tensor, Tensor], Optional[Exception]]): Tensor check function for strided tensors.
+    """
+
+    @functools.wraps(check_tensors)
+    def wrapper(actual: Tensor, expected: Tensor, **kwargs: Any) -> Optional[Exception]:
```

Comment:
We might want to do a holistic code structure review sometime (not in this pr); I'm a little concerned that these decorators aren't the best approach